packaging/rpm/dotnet-config.json

```diff
@@ -0,0 +1,41 @@
+{
+    "maintainer_name": "Microsoft",
+    "maintainer_email": "dotnetcore@microsoft.com",
+    "vendor": ".NET Foundation",
+    "package_name": "%SDK_RPM_PACKAGE_NAME%",
+    "install_root": "/usr/share/dotnet",
+    "install_doc": "/usr/share/doc/%SDK_RPM_PACKAGE_NAME%/",
+    "short_description": "%CLI_SDK_BRAND_NAME% %SDK_NUGET_VERSION%",
+    "long_description": ".NET Core is a development platform that you can use to build command-line applications, microservices and modern websites. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/dotnet/core). We happily accept issues and PRs.",
+    "homepage": "https://github.com/dotnet/core",
+    "release": {
+        "package_version": "1.0.0.0",
+        "package_revision": "1",
+        "urgency": "low",
+        "changelog_message": "Bootstrap loop package"
+    },
+    "control": {
+        "priority": "standard",
+        "section": "libs",
```

Comment:
RPM has a very different naming for libraries. `Development/Languages` and `Development/Libraries` are some valid options.